src/main/java/frc/robot/Robot.java

```diff
@@ -60,6 +60,7 @@ public void teleopInit() {
     if (m_autonomousCommand != null) {
       m_autonomousCommand.cancel();
     }
+    m_robotContainer.teleopInit();
```

Comment:
is this necessary? (can you explain why you added this) I would assume robotcontainer calls teleopInit itself 

(autonomousInit above doesnt call robotcontainer autonomousInit)

Reply:
`teleopInit` is a method I added to robot container. It starts all of the default commands on the subsystems (i.e. `DefaultDrive`, `IntakeNote`, etc). I do not want these commands running on the robot during auto, I would prefer to do everything manually because it reduces complexity.

Reply:
for some reason i thought robotcontainer extended something, but it doesn't. NVM